MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst LICENSE NOTICE HISTORY.rst requests/cacert.pem
+recursive-include tests *.py
```

Comment:
I created #4027 with graft/exclude-global. I leave it to up to you guys to pick one. :)